R/agglomerate.R

```diff
@@ -229,7 +229,7 @@ setMethod("agglomerateByRank", signature = c(x = "SummarizedExperiment"),
         # Remove those columns from rowData that include only NAs
         x <- .remove_NA_cols_from_rowdata(x, ...)
         x <- .add_values_to_metadata(x, "agglomerated_by_rank", rank)
-        x
+        x <- x[ order(rownames(x)), ]
```

Comment:
Good to add comment. What is this line doing? --> easier to debug and read the code afterwards

Reply:
ACK.